moped-database/config/hasura.local.yaml

```diff
@@ -4,3 +4,4 @@ metadata_directory: metadata
 actions:
   kind: synchronous
   handler_webhook_baseurl: http://localhost:3000
+admin_secret: hasurapassword
```

Comment:
the schemaspy github action uses this file, so this needs to be updated for the db docs to keep working

Reply:
i believe folks will also need to run a one-time ` ./hasura-cluster setenv local` to make sure their local `config.yaml` gets updated with this variable. it was enlightening to debug this with you @chiaberry!